packages/terra-clinical-item-view/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated Jest tests
+
```

Comment:
This is not needed as this change has no consumer value and it will essentially trigger a duplicate release.
```suggestion
```